eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -74,15 +70,6 @@ jobs:
           pip install doc-warden==$(DocWardenVersion)
           ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
         displayName: "Verify Readmes"
-      - task: UseDotNet@2
-        displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
-        inputs:
-          version: "$(DotNetCoreSDKVersion)"
-      - task: UseDotNet@2
-        displayName: "Use .NET Core runtime $(DotNetCoreRuntimeVersion)"
```

Comment:
I think we still need to set the runtime somehow

Reply:
Am not sure that the UseDotNetCore@2 task was specifying the runtime version. It only installs it and set it on the path. Since the runtime is already installed on the agent I don't think this was adding anything extra. I added it back for the runtime either ways, but the build runs fine without it.

Reply:
We should do some validation and because we are definitely trying to test against a specific runtime and if that isn't working we should fix that.

Reply:
It's also only a matter of time before the agents change what's installed by default. Being explicit would make sure when that happens we continue to work.